integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiAuxiliaryImage.java

```diff
@@ -733,40 +738,92 @@ void testCreateDomainWithConfigMapAndEmptryModelFileDir() {
                     .wdtModelHome(auxiliaryImagePathCustom + "/models");
     createAndPushAuxiliaryImage(MII_AUXILIARY_IMAGE_NAME,miiAuxiliaryImage12Tag, witParams);
 
-    String configMapName = "modelfiles-cm";
-    logger.info("Create ConfigMap {0} in namespace {1} with WDT models {3} and {4}",
-            configMapName, domainNamespace, MII_BASIC_WDT_MODEL_FILE, "/multi-model-one-ds.20.yaml");
-
     //create empty configMap with no models files and verify that domain creation failed.
     List<Path> cmFiles = new ArrayList<>();
 
+    String configMapName1 = "modelfiles1-cm";
+
+    //add model files to configmap and verify domain is running
+    cmFiles.add(
+            Paths.get(MODEL_DIR + "/" + MII_BASIC_WDT_MODEL_FILE));
+    cmFiles.add(Paths.get(MODEL_DIR + "/multi-model-one-ds.20.yaml"));
     assertDoesNotThrow(
             () -> createConfigMapForDomainCreation(
-                    configMapName, cmFiles, domainNamespace, this.getClass().getSimpleName()),
+                    configMapName1, cmFiles, domainNamespace, this.getClass().getSimpleName()),
             "Create configmap for domain creation failed");
-    OffsetDateTime timestamp = now();
 
     // create domain custom resource using auxiliary image
     logger.info("Creating domain custom resource with domainUid {0} and auxiliary image {1}",
             domainUid, miiAuxiliaryImage12);
-    final DomainResource domainCR = createDomainResourceWithAuxiliaryImage(domainUid, domainNamespace,
+    DomainResource domainCR1 = createDomainResourceWithAuxiliaryImage(domainUid, domainNamespace,
             WEBLOGIC_IMAGE_TO_USE_IN_SPEC, adminSecretName, createSecretsForImageRepos(domainNamespace),
             encryptionSecretName, replicaCount, auxiliaryImagePathCustom,
             miiAuxiliaryImage12);
+    assertNotNull(domainCR1, "failed to create domain resource");
+    domainCR1.spec().configuration().model().configMap(configMapName1);
+    createDomainAndVerify(domainUid, domainCR1, domainNamespace,
+            adminServerPodName, managedServerPrefix, replicaCount);
+  }
+
+  /**
+   * Create a domain using auxiliary image with empty configMap and no model files.
+   * Verify domain events and operator log contains the expected error message.
```

Comment:
Provide the description of expected message  (createDomain argument processing failed: createDomain failed to find a model file)